src/Detector/Golang/GolangDetector.cs

```diff
@@ -36,8 +36,8 @@ public PlatformDetectorResult Detect(DetectorContext context)
             // check if go.mod exists
             if (!sourceRepo.FileExists(GolangConstants.GoModFileName))
             {
-                _logger.LogDebug(
-                    $"Could not find {GolangConstants.GoModFileName} in repo");
+                _logger.LogError(
+                    $"Could not find {GolangConstants.GoModFileName} in repo, please add {GolangConstants.GoModFileName}");
```

Comment:
nit: Could not find {GolangConstants.GoModFileName} in root of the repo

Reply:
Updated